### PR TITLE
[Docs] Refactor `1_config.md`

### PR DESCRIPTION
**Pull request by Dai-Wenxun**
## Modification

Refactor `1_config.md`.

### CODE REVIEW

File: docs/en/user_guides/1_config.md [hunk 1]
```diff
@@ -103,15 +103,15 @@ which is convenient to conduct various experiments.
           consensus=dict(type='AvgConsensus', dim=1),  # Config of consensus module
           dropout_ratio=0.4,  # Probability in dropout layer
           init_std=0.01, # Std value for linear layer initiation
-          average_clips=None),
+          average_clips=None),  # Method to average multiple clip results
```
Comment (cir7) [inline docs/en/user_guides/1_config.md:106]: assign `average_clips` as `prob` is better?

Comment (Dai-Wenxun) [reply docs/en/user_guides/1_config.md]: done

File: docs/en/user_guides/1_config.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ which is convenient to conduct various experiments.
   ```python
   # model settings
   model = dict(  # Config of the model
-      type='Recognizer2D',  # Type of the recognizer
+      type='Recognizer2D',  # Name of the recognizer
```
Comment (ly015) [inline docs/en/user_guides/1_config.md:92]: What about 'Class name of the recognizer'?

Comment (Dai-Wenxun) [reply docs/en/user_guides/1_config.md:92]:
```suggestion
      type='Recognizer2D',  # Class name of the recognizer
```

File: docs/en/user_guides/1_config.md [hunk 3]
```diff
@@ -317,7 +318,8 @@ We incorporate modular design into our config system, which is convenient to con
   ```python
   # model setting
   model = dict(  # Config of the model
-      type='FastRCNN',  # Type of the detector
+      type='FastRCNN',  # Name of the detector
```
Comment (Dai-Wenxun) [inline docs/en/user_guides/1_config.md:321]:
```suggestion
      type='FastRCNN',  # Class name of the detector
```

File: docs/en/user_guides/1_config.md [hunk 4]
```diff
@@ -528,7 +549,7 @@ which is convenient to conduct various experiments.
   ```python
   # model settings
   model = dict(  # Config of the model
-      type='BMN',  # Type of the localizer
+      type='BMN',  # Name of the localizer
```
Comment (Dai-Wenxun) [inline docs/en/user_guides/1_config.md:552]:
```suggestion
      type='BMN',  # Class name of the localizer
```

**Review by cir7 — APPROVED** [review]

**Pull request merged**